ansible_collections/arista/avd/docs/contribution/semantic-versioning.md

```diff
@@ -0,0 +1,122 @@
+<!--
+  ~ Copyright (c) 2023 Arista Networks, Inc.
+  ~ Use of this source code is governed by the Apache License 2.0
+  ~ that can be found in the LICENSE file.
+  -->
+
+# Semantic Versioning
+
+The AVD project follows Semantic Versioning ([SemVer](https://semver.org/)).
+This document describes what attributes and APIs are stable and follow SemVer and which ones may change between minor releases.
+
+## Ansible Roles
+
+### eos_designs
+
+Inputs:
+
+- All role input variables follow SemVer.
+  - Any breaking changes will at a minimum be communicated with a deprecation notice for 6 months before being removed.
```

Comment:
It may be worthwhile to add this as a note at the start of the doc, it seems this is followed for each portion besides the Ansible plugins